config/policy.go

```diff
@@ -162,6 +162,11 @@ type Policy struct {
 	// SetResponseHeaders sets response headers.
 	SetResponseHeaders map[string]string `mapstructure:"set_response_headers" yaml:"set_response_headers,omitempty"`
 
+	// IDPClientID is the client id used for the identity provider.
+	IDPClientID string `mapstructure:"idp_client_id" yaml:"idp_client_id,omitempty"`
```

Comment:
Flagging @alexfornuto for docs requirement.

Reply:
There is another issue for updating docs, but sure I can include those changes in this PR.

Reply:
I missed that.